client/components/data/query-reader-feeds-search/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestFeedSearch } from 'state/reader/feed-search/actions';
+
+class QueryFeedSearch extends Component {
+	static propTypes = {
+		query: PropTypes.string,
+		searchFeeds: PropTypes.func,
+	}
+
+	componentWillMount() {
```

Comment:
Its fishy that I haven't run into that issue in my connecting-ui pr. I'll update that here.